test/services/publisher_transactions_getter_test.rb

```diff
@@ -1,34 +1,34 @@
 require "test_helper"
 
 class PublisherBalanceGetterTest < ActiveJob::TestCase
+  describe "mocked transactions" do
+    before(:example) do
+      @prev_offline = Rails.application.secrets[:api_eyeshade_offline]
+      Rails.application.secrets[:api_eyeshade_offline] = false
```

Comment:
Setting this here will affect other tests and can lead to random failures, can we find a way to limit this change to this test only? I'd prefer something injectable, but in a pinch[ a stub](https://www.rubydoc.info/gems/minitest/4.2.0/Object:stub) should work.

Reply:
The lines below

```   
after(:example) do
      Rails.application.secrets[:api_eyeshade_offline] = @prev_offline
    end
```

will reset the state